### PR TITLE
Meta: enable preview and diffs for all pull requests

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

File: .pr-preview.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "src_file": "url.bs",
+    "ls_url": "https://url.spec.whatwg.org/",
```
Comment (tobie) [inline .pr-preview.json]: You want to make that "ed_url" instead.

File: .pr-preview.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "title": "URL Standard",
+    "back_to_ls_link": "<a href=\"https://url.spec.whatwg.org/\" id=\"commit-snapshot-link\">Go to the living standard</a>",
+    "bikeshed_parameters": {
+        "force": 1,
+        "md-status": "LS-COMMIT",
+        "md-h1": "URL <small>(<a href=\"{{ pull_request.html_url }}\">PR #{{ pull_request.number }}</a>)</small>",
+        "md-warning": "Commit {{ short_sha }} {{ pull_request.head.repo.html_url }}/commit/{{ sha }} replaced by {{ config.ls_url }}",
```
Comment (tobie) [inline .pr-preview.json]: s/config.ls_url/config.ed_url/

**Comment by annevk** [comment]
Note that I copied this straight from your example...

**Comment by tobie** [comment]
Yes. I know. I just fixed it.

**Pull request merged**